css-three-column-layout/styles.css

```diff
@@ -0,0 +1,60 @@
+* {
+  font-family: sans-serif;
+  color: white;
+  font-weight: 300;
+}
+
+.background {
+  background-color: rgb(26, 150, 217);
+}
+
+.container {
+  max-width: 850px;
+  max-height: 600px;
```

Comment:
For the most part, you don't want to set both a `width` & a `height`. It's usually one or the other. I think if you remove the `max-height` it should be fine.